src/hooks/useExcludeInterest.js

```diff
@@ -0,0 +1,83 @@
+import { useEffect, useState } from "react";
+
+export default function useExcludeInterest(api) {
```

Comment:
This name is confusing for me. What would be a better name? Maybe something like `useLearnerContentPreferences`?

Reply:
What about renaming it to, for instance, `useUnwantedContentPreferences` or `useLearnerFilterPreferences`?  I think these examples indicate more that the hook is meant to exclude content. What do you think?

Reply:
Ah, it seems taht I don't fully understand what this hook does. 
In TagsOfFilters I see: 

```
  const {
    availableFilters,
    toggleFilterSubscription,
    isSubscribedSearchFilter,

    subscribedSearchFilters,
    subscribeToSearchFilter,
    removeSearchFilter,

    showModal,
    setShowModal,
  } = useExcludeInterest(api);

``` 

This seems to be more than excluded preferences, or? `availableFilters` at least sounds like ... ah, it's not `availableTopics`. It's `topicsAvailableForFiltering`. Right?

Reply:
In this case `useUnwantedContentPreferences` is good. And maybe that `availableFilters` could really have been renamed to `toipcsAvailableForFiltering` all the way from `ZeeguuAPI` into the hook.

Reply:
✅ Renaming suggestions are now applied

Reply:
Tak @igawaclawska !